### PR TITLE
Fixes responsiveness in Facilty Home

### LINKED ISSUE #6621: Make the facility detail page more properly responsive
**Describe the bug**

- [ ] the facility details which can be shown in the same row are been displayed as a different row in the responsive view
- [ ] the buttons are stretched a lot to occupy the entire row, where it can occupy two or three buttons as per the screen size 

**To Reproduce**

1. Go to facility detail page
2. See error

**Expected behavior**

- [ ] make the facility detail page, fully responsive properly

**Screenshots**

image

### PR DESCRIPTION
**Pull request by konavivekramakrishna**
## Proposed Changes

- Fixes #6621 
https://www.loom.com/share/fa9b9e8069e84ffdbba3a13d39c397dc?sid=fcd8ab0b-41f6-48aa-ba2a-ae50690c903a

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by vivramkrishna** [comment]
image

Collapsing the sidebar to prevent overlapping with buttons caused by long addresses.

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -123,6 +123,7 @@ export const FacilityHome = (props: any) => {
       title={facilityData?.name || "Facility"}
       crumbsReplacements={{ [facilityId]: { name: facilityData?.name } }}
       focusOnLoad={true}
+      collapseSidebar
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx:126]: Was this intended?

```suggestion
```

**Comment by nihal467** [comment]
@konavivekramakrishna 

- [ ] Don't keep the sidebar by default collapsed for the facility detail page in the normal view

image

- [ ] if there any overflow or overlapping, just take it to next line

**Comment by vivramkrishna** [comment]
https://www.loom.com/share/af7217bb652e4d7696ca417d78deec5d?sid=c399a179-7f41-42ec-9834-489d588c492d

@nihal467 can you please check once

**Comment by vivramkrishna** [comment]
@nihal467 

can you please check it
I have changed accordingly

File: src/Components/Facility/FacilityHome.tsx [hunk 2]
```diff
@@ -222,8 +222,8 @@ export const FacilityHome = (props: any) => {
                 </div>
               </div>
               <div className="flex flex-1 items-center">
-                <div className="mb-6 grid  w-full grid-cols-1 gap-4 md:mb-0 lg:grid-cols-2">
-                  <div className="flex-col justify-between md:flex lg:flex-1 ">
+                <div className="mb-6 grid  w-full   gap-4 sm:grid-cols-2   md:mb-0  lg:grid-cols-2">
+                  <div className="grid flex-col justify-between md:flex lg:flex-1 ">
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx]: You've used `grid` without specifying `grid-cols-{x}`, but with `flex-col`. Why? What does that try to achieve?

File: src/Components/Facility/FacilityHome.tsx [hunk 3]
```diff
@@ -242,7 +242,7 @@ export const FacilityHome = (props: any) => {
                       </div>
                     </div>
                   </div>
-                  <div className="min-w-[300px] flex-col md:flex lg:flex-1">
+                  <div className="  flex-col md:flex lg:flex-1">
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx]:
```suggestion
                  <div className="flex-col md:flex lg:flex-1">
```

**Comment by nihal467** [comment]
LGTM

**Comment by khavinshankar** [comment]
@rithviknishad @Ashesh3 please review this pr

File: src/Components/Facility/FacilityHome.tsx [hunk 4]
```diff
@@ -222,7 +222,7 @@ export const FacilityHome = (props: any) => {
                 </div>
               </div>
               <div className="flex flex-1 items-center">
-                <div className="mb-6 grid  w-full grid-cols-1 gap-4 md:mb-0 lg:grid-cols-2">
+                <div className="mb-6 grid  w-full   gap-4 sm:grid-cols-2   md:mb-0  lg:grid-cols-2">
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx]:
```suggestion
                <div className="mb-6 grid w-full gap-4 sm:grid-cols-2 md:mb-0 lg:grid-cols-2">
```

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Facility/FacilityHome.tsx (hunk 4 above)
Comment (Ashesh3) [inline src/Components/Facility/FacilityHome.tsx]:
```suggestion
                <div className="mb-6 grid w-full gap-4 sm:grid-cols-2 md:mb-0 lg:grid-cols-2">
```

**Review by Ashesh3 — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**